client/src/components/Allocations/AllocationsModals/AllocationsManageTeamModal/AllocationsManageTeamModal.js

```diff
@@ -8,21 +8,21 @@ import './AllocationsManageTeamModal.module.scss';
 
 export const AllocationsManageTeamTable = ({ rawData, projectId }) => {
   const dispatch = useDispatch();
-  const { removingUserOperation } = useSelector(state => state.allocations);
```

Comment:
looks like we need to remove this file from the main 737 branch, but could you remove it here in the meantime?